addons/website/views/snippets/s_tabs.xml

```diff
@@ -7,13 +7,13 @@
             <div class="s_tabs_main">
                 <div class="s_tabs_nav mb-3">
                     <ul class="nav nav-pills" role="tablist">
-                        <li class="nav-item">
+                        <li class="nav-item" role="presentation">
```

Comment:
I am curious: what differentiates this from a menu?

Reply:
I am not sure to understand your question but here, the `role="presentation"` is needed as the anchor with `role="tab"` is a child of the `li` element.